exercises/series/src/Example.hs

```diff
@@ -4,5 +4,6 @@ import Data.Char (digitToInt)
 import Data.List (tails)
 
 slices :: Int -> String -> [[Int]]
+slices 0 _ = [[]]
```

Comment:
Or how about I make a crazy suggestion! We write the tests to accept either `[[]]` or `[[], [], []]` for `slices 0 "012"` !